static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js

```diff
@@ -656,18 +657,6 @@ describe('triplelift adapter', () => {
         });
     });
 
-    test('should return correct triplelift adapter params for leaderboard', () => {
-        containsLeaderboard.mockReturnValueOnce(false);
-        containsMpu.mockReturnValueOnce(false);
-        containsDmpu.mockReturnValueOnce(true);
-        containsMobileSticky.mockReturnValueOnce(false);
-
-        const tripleLiftBids = bids('dfp-right', [[300, 600]])[1].params;
-        expect(tripleLiftBids).toEqual({
-            inventoryCode: 'theguardian_article_300x600_prebid',
-        });
```

Comment:
Why was this removed in the end?

Reply:
Oh strange! The diff it showed me was this being removed!

Reply:
yes i deleted it along with adding an extra condition on mpu slot. We only have an **article** inventoryCode for mbu so I have added isArticle condition on it. Tried to fix the test but the way it's set up it's very hard to mock it only for that particular test. The isArticle variable gets initialised the same time with the imports and even changing it later on it doesn't change. I will merge it for now and see if I can add this test back by re-structuring these tests on a subsequent PR